Makefile

```diff
@@ -60,14 +63,18 @@ setup-git-config:
 setup-git: ensure-venv setup-git-config
 	@echo "--> Installing git hooks"
 	cd .git/hooks && ln -sf ../../config/hooks/* ./
+	@# This 3.x.x version needs to be synced with the version in .python-version.
+	@PYENV_VERSION=3.6.7 python3 -c '' || (echo 'Please run `make setup-pyenv` to install the required Python 3 version.'; exit 1)
```

Comment:
Is there any way for us to keep this in sync? I imagine this will break at some point.

Reply:
https://github.com/getsentry/sentry/pull/18906/commits/7e3b8b2ced4cdd47933d98a9d147d2e3de93056e